mix.exs

```diff
@@ -60,20 +62,30 @@ defmodule MMDB2Decoder.MixProject do
 
   defp docs do
     [
-      main: "MMDB2Decoder",
+      extras: [
+        "CHANGELOG.md",
+        LICENSE: [title: "License"],
+        "README.md": [title: "Overview"]
+      ],
+      main: "readme",
       source_ref: "master",
-      source_url: @url_github
+      source_url: @url_github,
+      api_reference: false,
```

Comment:
```suggestion
```

Or is there a good reason to not generate that page?

Reply:
With:
![image](https://user-images.githubusercontent.com/134518/175781262-9e2aca2d-f9db-468e-a5a1-daff8fcf5637.png)

Without:
![image](https://user-images.githubusercontent.com/134518/175781301-dc65a449-82a8-42ca-bb31-44506eff2836.png)

The `Modules` link under `API References` already duplicated on the `Modules` on top navigation.

Reply:
As the entry to the documentation opens on `Modules` I would say we just keep it. Should not hurt to have that default page around.